### PR TITLE
add error message on load old capture and warning on save

### PR DESCRIPTION
**Pull request by IrinaShkviro**
Added warning on saving capture and updated error on loading capture
capture_on_save_warning

capture_on_load_error

### CODE REVIEW

**Comment by beckerhe** [comment]
I think the message is not quite spot on yet. But I leave it to @ronaldfw to make a suggestion since he is way better than me in crafting a good English message.

File: OrbitQt/orbitmainwindow.cpp [hunk 1]
```diff
@@ -705,7 +708,30 @@ void OrbitMainWindow::on_actionHelp_triggered() { GOrbitApp->ToggleDrawHelp(); }
 
 void OrbitMainWindow::on_actionFeedback_triggered() { ShowFeedbackDialog(); }
 
+void OrbitMainWindow::ShowCaptureOnSaveWarningIfNeeded() {
+  const QString skip_capture_warning =
+      QString::fromStdString("SkipCaptureVersionWarning");
+  if (settings.value(skip_capture_warning, 0).toInt() == 0) {
+    QMessageBox message_box;
+    message_box.setText(
+        "Note: Old captures might be unavailable in the future versions. "
```
Comment (ronaldfw) [inline OrbitQt/orbitmainwindow.cpp:717]: Suggested wording:
"Note: Captures saved with this version of Orbit might be incompatible with future versions. Please check release notes for more information"

This makes it clear that captures saved with the current version might become incompatible. Also, fix wording, "unavailable" doesn't really sound right to me.

File: OrbitQt/orbitmainwindow.cpp [hunk 2]
```diff
@@ -743,10 +769,13 @@ outcome::result<void> OrbitMainWindow::OpenCapture(
...
     SetTitle({});
-    QMessageBox::critical(this, "Error loading capture",
-                          QString::fromStdString(absl::StrFormat(
-                              "Could not load capture from \"%s\":\n%s.",
-                              filepath, result.error().message())));
+    QMessageBox::critical(
+        this, "Error loading capture",
+        QString::fromStdString(absl::StrFormat(
+            "Could not load capture from \"%s\":\n%s.\nNote: If capture was "
+            "taken in previous Orbit version, it could be unavailable. Please "
```
Comment (ronaldfw) [inline OrbitQt/orbitmainwindow.cpp:776]: Nit: "taken in a previous Orbit version" (add "a"). 

And change "unavailable" to "incompatible".

File: OrbitQt/orbitmainwindow.cpp [hunk 3]
```diff
@@ -743,10 +769,13 @@ outcome::result<void> OrbitMainWindow::OpenCapture(
...
+            "check release notes for information.",
```
Comment (ronaldfw) [inline OrbitQt/orbitmainwindow.cpp:777]: "more information"

File: OrbitQt/orbitmainwindow.cpp [hunk 4]
```diff
@@ -743,10 +769,13 @@ outcome::result<void> OrbitMainWindow::OpenCapture(
...
   if (result.has_error()) {
     SetTitle({});
-    QMessageBox::critical(this, "Error loading capture",
-                          QString::fromStdString(absl::StrFormat(
-                              "Could not load capture from \"%s\":\n%s.",
-                              filepath, result.error().message())));
+    QMessageBox::critical(
+        this, "Error loading capture",
+        QString::fromStdString(absl::StrFormat(
+            "Could not load capture from \"%s\":\n%s.\nNote: If capture was "
```
Comment (ronaldfw) [inline OrbitQt/orbitmainwindow.cpp:775]: Nit: "If the capture was..." (add "the").

File: OrbitQt/orbitmainwindow.cpp [hunk 5]
```diff
@@ -705,7 +708,30 @@ void OrbitMainWindow::on_actionHelp_triggered() { GOrbitApp->ToggleDrawHelp(); }
 
 void OrbitMainWindow::on_actionFeedback_triggered() { ShowFeedbackDialog(); }
 
+void OrbitMainWindow::ShowCaptureOnSaveWarningIfNeeded() {
+  const QString skip_capture_warning =
+      QString::fromStdString("SkipCaptureVersionWarning");
+  if (settings.value(skip_capture_warning, 0).toInt() == 0) {
```
[the remark this answers is not in the document]
Comment (beckerhe) [reply OrbitQt/orbitmainwindow.cpp:714]: And there should also be support for a "bool" (instead of an int).

**Comment by IrinaShkviro** [comment]
> Some suggestions on how to change the wording, but o/w LGTM

Thanks a lot! Updated the messages.

File: OrbitQt/orbitmainwindow.cpp (hunk 5 above)
[the remark this answers is not in the document]
Comment (IrinaShkviro) [reply OrbitQt/orbitmainwindow.cpp]: Thanks, fixed it: changed to bool, saved this setting in registry (organization: "The Orbit Authors",  application: "Orbit Profiler").

**Comment by beckerhe** [comment]
Thanks!

**Pull request merged**